src/views/components-wallet/TheWalletTopRightText.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="mew-component--top-right-text">
+    <v-skeleton-loader v-if="loading" type="text" width="120px" />
+    <div v-else class="text">{{ text }}</div>
+  </div>
+</template>
+
+<script>
+export default {
+  components: {},
+  props: {
+    loading: {
+      type: Boolean,
+      default: false
+    },
+    text: {
```

Comment:
- rename text prop, to balance
- implement computed  texts as: 
return string  --> `'${$t('balance')}: ${this.balance}'` . Where $t('balance') is the balance string to translate, and this.balance is a balance prop

Reply:
fixed